### PR TITLE
Match NTSC z_kaleido_scope.c

### PR DESCRIPTION
**Pull request by cadmic**
Unfortunately `LANGUAGE_ARRAY` doesn't work here. There's some more REG-shuffling going on here, but tbh I don't really understand the code here so named them all `UNK`. Maybe someone has name ideas?

### CODE REVIEW

**Comment by Dragorn421** [comment]
Could you use the macros for newly named regs everywhere? For example in z_construct.c where they're initialized

This seems related to #1987 so probably merge that other PR first to be able to do that?

**Comment by cadmic** [comment]
Yeah good point. I think I can still do it now and deal with the merge conflicts later once one of the PRs wins

In PAL versions, YREG(48) and friends still get initialized in Regs_InitDataImpl even though the named version has been ifdef'd out. I added comments for these but is there something else we can do here?

**Comment by Dragorn421** [comment]
Looks good to me but waiting for #1987 about `Regs_InitDataImpl` comments

File: src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope.c [hunk 1]
```diff
@@ -132,18 +132,49 @@ static void* sGameOverTexs[] = {
     gPauseSave20Tex,     gPauseSave21Tex, gPauseSave22Tex, gPauseSave23Tex, gPauseSave24Tex,
 };
 
-static void* sEquipmentTexs[] =
-    LANGUAGE_ARRAY(sEquipmentJPNTexs, sEquipmentENGTexs, sEquipmentGERTexs, sEquipmentFRATexs);
+#if OOT_NTSC
+#define EQUIPMENT_TEXS(language) ((language) != LANGUAGE_JPN ? sEquipmentENGTexs : sEquipmentJPNTexs)
+#define SELECT_ITEM_TEXS(language) ((language) != LANGUAGE_JPN ? sSelectItemENGTexs : sSelectItemJPNTexs)
+#define MAP_TEXS(language) ((language) != LANGUAGE_JPN ? sMapENGTexs : sMapJPNTexs)
```
Comment (fig02) [inline src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope.c:138]: this isnt due to your changes, just want to take the opportunity that I think "texs" is awkward to read and say. it should prob just say the whole word "textures" lol

File: src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope.c [hunk 2]
```diff
@@ -3335,6 +3375,9 @@ void KaleidoScope_Update(PlayState* play) {
                             WREG(2) = -6240;
                             YREG(8) = pauseCtx->unk_204;
                             func_800F64E0(0);
+#if OOT_NTSC
+                            AudioOcarina_SetInstrument(OCARINA_INSTRUMENT_OFF);
+#endif
```
Comment (fig02) [inline src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope.c:3380]: nit: extra line break that isnt present in the one above

**Review by fig02 — APPROVED** [review]

File: src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope.c (hunk 2 above)
Comment (fig02) [reply src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope.c:3380]: nvm I got tricked by the way github was formatting it

File: src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope.c (hunk 1 above)
Comment (cadmic) [reply src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope.c:138]: Hm I was about to change this, but there's actually quite a lot of arrays called `Texs` already. There's no precedent for the macros though, should I change those?

Comment (fig02) [reply src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope.c:138]: no it should probably just be changed all at once, if people agree with my sentiment lol

**Review by Dragorn421 — APPROVED** [review]

File: src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope.c (hunk 1 above)
Comment (Dragorn421) [reply src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope.c:138]: I like Texs

Comment (fig02) [reply src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope.c:138]: I would compromise with `Texes` I guess, but pluralizing a word ending in x with just s is weird to look at (and wrong)
But I also dont think texture is much longer/compromising readability

Comment (Dragorn421) [reply src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope.c:138]: I'm a weirdo who will name a 2d array of an already plural something `thingsss`
as in
```py
thingsss = []
for i in range(5):
  thingss = []
  for j in range(5):
    things = Things()
    thingss.append(things)
  thingsss.append(thingss)

for thingss in thingsss:
  for things in thingss:
    print(things)
```

**Pull request merged**